src/scss/_components.percentage-card.scss

```diff
@@ -0,0 +1,54 @@
+.cmp-percentage-card-grid {
+  @supports (display: grid) {
```

Comment:
For the needs of this layout, Flexbox is probably sufficient. If I got you going down the path of CSS Grid on this one, my apologies. No worries if you want to leave Grid, not a big deal, I would recommend adding in Flexbox outside of `@supports` for a fallback.

Reply:
Switched to Flexbox for this whole component, good call.